actors/miner/src/beneficiary.rs

```diff
@@ -0,0 +1,87 @@
+use fvm_ipld_encoding::tuple::*;
+use fvm_ipld_encoding::Cbor;
+use fvm_shared::address::Address;
+use fvm_shared::bigint::bigint_ser;
+use fvm_shared::clock::ChainEpoch;
+use fvm_shared::econ::TokenAmount;
+use std::cmp::max;
+
+#[derive(Debug, PartialEq, Serialize_tuple, Deserialize_tuple)]
+pub struct BeneficiaryTerm {
+    // Quota: The total amount the current beneficiary can withdraw. Monotonic, but reset when beneficiary changes.
```

Comment:
Use `///` rustdoc comments (and then it's probably not necessary to repeat the field's name)